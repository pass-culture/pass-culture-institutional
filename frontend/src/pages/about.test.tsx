import React from 'react'
import { describe, expect, it } from 'vitest'
import { axe } from 'vitest-axe'

import About from './about'
import { render } from '@/tests'

describe('About page', () => {
  it('should pass axe accessibility tests', async () => {
    const { container } = render(<About />)

    expect(await axe(container)).toHaveNoViolations()
  })

  it('should render the page', () => {
    const { container } = render(<About />)

    expect(container).toMatchSnapshot()
  })

})
