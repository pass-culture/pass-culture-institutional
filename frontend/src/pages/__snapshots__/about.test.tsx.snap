// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`About page > should render the page 1`] = `
<div>
  <div
    class="sc-gtsqUy qFqbA"
  >
    <main
      class="sc-bdnyFh cOrhoO"
    >
      <h1
        class="sc-gKAaef ljqyJc"
      >
        About Page
      </h1>
      <p
        class="sc-iCoHzw jGcKYw"
      >
        Get started by editing
        <code
          class="sc-jSFipO cMRfmt"
        >
          pages/about.tsx
        </code>
      </p>
      <h1
        class="sc-gKAaef ljqyJc"
      >
        Lorem ipsum dolor sit amet
      </h1>
      <p
        class="sc-iCoHzw jGcKYw"
      >
        Lorem ipsum dolor sit amet, consectetur adipiscing elit. Nulla euismod, nisl eget ultricies ultrices, nunc nisl aliquam nunc, vitae aliquam nisl nunc eu
      </p>
      <div
        class="sc-dlniIP RElFz"
      >
        <div
          class="sc-hKFymg gVqXtY"
        >
          <a
            class="sc-eCAqax jVKTjr"
            href="/"
          >
            ‚Üê Go Back
          </a>
        </div>
      </div>
    </main>
  </div>
</div>
`;
